dashboard/lib/services/lti/account_linker.rb

```diff
@@ -0,0 +1,23 @@
+module Services
+  module Lti
+    class AccountLinker < Services::Base
+      def initialize(user:, session:)
+        @user = user
+        @session = session
+      end
+
+      def call
+        rehydrated_user = ::User.new_with_session(ActionController::Parameters.new, @session)
+        auth_option = rehydrated_user.authentication_options.first
+        ActiveRecord::Base.transaction do
+          @user.authentication_options << auth_option
+          Services::Lti.create_lti_user_identity(@user)
+          if @user.user_type == ::User::TYPE_TEACHER
+            @user.lti_roster_sync_enabled = true
+          end
+          PartialRegistration.delete(@session)
+        end
+      end
```

Comment:
https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#attr-family
```suggestion
      attr_reader :user, :session

      def initialize(user:, session:)
        @user = user
        @session = session
      end

      def call
        ActiveRecord::Base.transaction do
          user.authentication_options << rehydrated_user.authentication_options.first
          Services::Lti.create_lti_user_identity(user)
          user.lti_roster_sync_enabled = true if user.teacher?
          PartialRegistration.delete(session)
        end
      end

      private def rehydrated_user
        @rehydrated_user ||= ::User.new_with_session(ActionController::Parameters.new, session)
      end
```